lib/ember_cli/build_monitor.rb

```diff
@@ -32,7 +32,7 @@ def wait!
     attr_reader :name, :paths
 
     def complete?
-      !paths.lockfile.exist?
+      !paths.lockfile.exist? && paths.dist.join("index.html").exist?
```

Comment:
Yes, I do. My rack-timeout logic looks like this (in a Rails initializer):

    if Rails.env.development? or Rails.env.test?
      Rack::Timeout.timeout = 60
    else
      Rack::Timeout.timeout = 15  # seconds
    end

With the newest non-Sprockets builds it never reaches anywhere near 15 seconds of initial load time. It's more like 5 seconds on average right now.